src/interfaces/token.ts

```diff
@@ -118,6 +120,11 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
       }
     }
 
+    // TODO
+    if (250 === this.chainId) {
+      balances.fantom = [];
```

Comment:
Also not 100% sure what we should be doing here, any ideas?

Reply:
since fantom is the native token, we will need something like this to fetch it:
```
let balance;
if (tokenAddress === NATIVE_TOKEN_ADDRESS) {
    balance = await this.provider
        .getBalance(walletAddress)
        .then((result) => ({
            balance: result.toString(),
        }));
} else {
    const erc20Contract = getContract(
        tokenAddress,
        erc20.abi,
        this.provider
    );
    balance = await erc20Contract
        .balanceOf(walletAddress)
        .then((result) => ({
            balance: result.toString(),
        }));
}
```

Later we can refactor the erc20 balance fetch to a better solution like directly calling the BalancesHelper contract we have on mainnet https://etherscan.io/address/0x855ffe28019106d089bC018Df18838f8d241c402#readContract , since it accepts calling multiple addresses at once.

Reply:
We can chat a bit more about this, got it started with https://github.com/yearn/yearn-sdk/pull/262/commits/48e35da18ffbb87059e054c2dc697a6b6c94d6cc but still unsure what exactly needs to be done